### PR TITLE
Adding support for questions

### PR DESCRIPTION
**Pull request by Siggnja**
- Adds support for questions(each question gives a score of 10 points)
- Updates the rules to reflect this support
- Adds question support for players and add two questions for top scorer and golden glove
- Adds support in our scoring so that once we add the answer in our DB it will be automatically updated everywhere
- Moves questions to a seperate tab
- Adds support to click on a user in the league view to see their questions

**Screenshots**
Screenshot 2022-11-15 at 00 34 02
image

### CODE REVIEW

File: db/players.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
+const players = [
+  "Harry Kane",
```
Comment (Siggnja) [inline db/players.ts:2]: This is the top 143 players according to bookies.

File: db/players.ts [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  "Alphonso Davies",
+  "Jason Cummings",
+  "Iliman Ndiaye",
+  "Aaron Ramsey",
+  "Jewison Bennette",
+  "Each Way Terms"
+];
+
+export const goalies = [
+  "Alisson",
```
Comment (Siggnja) [inline db/players.ts:149]: Top 25 goalies according to bookies

File: app/pages/leagues/[id].tsx [hunk 3]
```diff
@@ -234,7 +253,7 @@ const LeaguesPage: BlitzPage = () => {
         <title>League</title>
       </Head>
 
-      <Container paddingTop="16px">
+      <Container paddingTop="16px" maxW="75ch">
```
[the remark this answers is not in the document]
Comment (sigginjals) [reply app/pages/leagues/[id].tsx:263]: I just saw that chakra-ui uses it by default 🥼 .
First time for me.

File: app/leagues/queries/getLeague.ts [hunk 4]
```diff
@@ -44,8 +44,28 @@ export default resolver.pipe(resolver.zod(GetLeague), resolver.authorize(), asyn
...
+
   if (!league) throw new NotFoundError()
 
+  const questionsForUsers = (await db.quizQuestion.findMany({
+    include: {
+      UserQuizQuestion: true,
+      translations: true,
+    },
+  })).map(q => ({ ...q, UserQuizQuestion: q.UserQuizQuestion.filter(a => usersInLeague.includes(a.userId)) }))
+
```
[the remark this answers is not in the document]
Comment (Siggnja) [reply app/leagues/queries/getLeague.ts:61]: @gunnartorfis Much better, I just couldn't figure out the syntax.

**Review by gunnartorfis — APPROVED** [review]
Boom! 🚀

**Pull request merged**